{
    "crime": "5614. Ljava/net/ProtocolException;<init>->Ljava/lang/AssertionError;getCause",
    "permission": [],
    "api": [
        {
            "class": "Ljava/net/ProtocolException;",
            "method": "<init>",
            "descriptor": "(Ljava/lang/String;)V"
        },
        {
            "class": "Ljava/lang/AssertionError;",
            "method": "getCause",
            "descriptor": "()Ljava/lang/Throwable;"
        }
    ],
    "score": 1
}